src/ontology/edits/oeo-physical.omn

```diff
@@ -1802,25 +1813,39 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/306",
 Class: OEO_00000181
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "Gas/diesel oil is an oil and petroleum product that is primarily a medium distillate distilling between 180 °C and 380 °C. Includes blending components. Several grades are available depending on uses."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Gas/diesel oil is a portion of matter that is primarily a medium distillate distilling between 180 °C and 380 °C. Includes blending components. Several grades are available depending on uses."@en,
```

Comment:
Is `gas diesel oil` the correct lable?? The def should use the correct lable, too.

Reply:
I did not change the label. The label was originally `GasDieselOil` before [changing to numeric identifiers](https://github.com/OpenEnergyPlatform/ontology/commit/ab041f6305e2ebdbc56ea7a424398977cca400b6#),

@jannahastings : Is there a problem if a label contains a slash?

Reply:
Okay, then I suggest, we keep the label for the moment as it is. This PR solves #811 only partially and we have to discuss this class later again anyway.